src/test-run/commands/options.js

```diff
@@ -6,35 +6,35 @@
 import Assignable from '../../utils/assignable';
 
 import {
-    ActionNumberOptionError,
-    ActionPositiveNumberOptionError,
+    ActionIntegerNumberOptionError,
+    ActionPositiveIntegerNumberOptionError,
     ActionBooleanOptionError
 } from '../../errors/test-run';
 
 // Validators
-function number (option, val, positive) {
+function integerNumber (option, val, positive) {
     var valType   = typeof val;
-    var ErrorCtor = positive ? ActionPositiveNumberOptionError : ActionNumberOptionError;
+    var ErrorCtor = positive ? ActionPositiveIntegerNumberOptionError : ActionIntegerNumberOptionError;
 
     if (valType !== 'number')
         throw new ErrorCtor(option, valType);
 
-    if (isNaN(val))
-        throw new ErrorCtor(option, valType);
+    if (isNaN(val) || !Number.isInteger(val))
```

Comment:
`Number.isInteger` not available in IE and Safari. It's ES2017 proposal. Use this polyfill instead:
```js
var isInteger =  typeof value === 'number'
                        && isFinite(value)
                        && Math.floor(value) === value;
```